api/src/main/scala/org/ensime/api/incoming.scala

```diff
@@ -53,7 +53,7 @@ final case class TypecheckFileReq(fileInfo: SourceFileInfo) extends RpcAnalyserR
 /**
  * Response with a `VoidResponse`.
  */
-final case class UnloadModuleReq(module: String) extends RpcAnalyserRequest
+final case class UnloadModuleReq(moduleId: EnsimeProjectId) extends RpcAnalyserRequest
```

Comment:
these changes to the API will show up in the wire protocol. You'll need to add a custom marshaller for `EnsimeProjectId` in jerk(y) and swank(y), e.g. see how we add a custom `DebugThreadIdFormat` (you'll need to add that custom formatter in 3 places, and it wouldn't be a bad idea to add tests)